test/DynamoCoreWpfTests/SerializationTests.cs

```diff
@@ -854,7 +855,7 @@ private void CompareWorkspaceViews(WorkspaceViewComparisonData a, WorkspaceViewC
                 var valueB = b.NodeViewDataMap[kvp.Key];
                 Assert.AreEqual(valueA, valueB,
                 string.Format("Node View Data:{0} value, {1} is not equal to {2}",
-                a.NodeViewDataMap[kvp.Key].Name, valueA, valueB));
+                a.NodeViewDataMap[kvp.Key].Name, JsonSerializer.Serialize(valueA), JsonSerializer.Serialize(valueB)));
```

Comment:
can this possibly work for all cases? I think it's fragile because we never know what test graphs are going to run.

Reply:
I did find it useful for investigating some parallel tests. Ex https://master-5.jenkins.autodesk.com/job/Dynamo/job/DynamoSelfServe/job/test_parallel_selfserve/87/testReport/junit/DynamoCoreWpfTests/JSONSerializationTests/SerializationTest__C___Jenkins__workspace__Dynamo__DynamoSelfServe__test_parallel_selfserve__Dynamo__test__core__dsevaluation__regress722_dyn__/
Not sure how much to invest in better visibility here.
I though I would keep it since it does not seem that it would break stuff. (but I can remove it if you think it is dangerous)